config/routes.rb

```diff
@@ -140,7 +140,7 @@
         resources :facts, :only => :index, :controller => :fact_values
         resources :puppetclasses, :only => :index
 
-        get 'parent_facts/:parent_fact/facts', :to => 'fact_values#index', :as => 'parent_fact_facts', :parent_fact => /[\w.:_-]+/
+        get 'parent_facts/:parent_fact/facts', :to => 'fact_values#index', :as => 'parent_fact_facts', :parent_fact => /[\/\w.:_-]+/
```

Comment:
I've found https://github.com/rails/rails/issues/16058, that - if we want to use '/' in a path parameter - to use a glob param (i.e. *id instead of :id).
```suggestion
        get 'parent_facts/*parent_fact/facts', :to => 'fact_values#index', :as => 'parent_fact_facts', :parent_fact => /[\/\w.:_-]+/
```